enviroment_scripts/redis_rq/start_redis_rq.sh

```diff
@@ -0,0 +1,3 @@
+sudo docker run -d -p $REDIS_PORT:6379 -v $REDIS_DATA:/data redis
+nohup rq-dashboard -P $REDIS_PORT &
+python3 test_redis_connection.py
```

Comment:
@orionw this PR looks good. What do you think about adding some comments to this script that gives context, like saying where to run this from, what it does, or how to access the rq-dashboard? Anything that's relevant and provides context as to what this script should be used for or how to use its output. Things like that.

If you have plans to add that documentation elsewhere, like when you document how to run the experimenter as a part of #49, then let me know and I can merge this PR. In general I would say document processes in whatever way you think would be easiest for a user of this repo to consume.

Reply:
I'll add it to the README.md once I go through the whole process.

I added documentation to the file though, for ease of use.  Let me know if I'm missing something.